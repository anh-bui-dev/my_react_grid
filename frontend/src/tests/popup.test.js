import { shallow, mount } from 'enzyme';
import React, { Component } from 'react';
import Popup from '../components/popup';

describe('Popup', () => {
  it('Should render correctly', () => {
    const wrapper = mount(<Popup />);
    expect(wrapper).toMatchSnapshot();
  });
});