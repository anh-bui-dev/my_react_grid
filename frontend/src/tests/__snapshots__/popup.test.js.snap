// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Popup Should render correctly 1`] = `
<Popup>
  <div
    id="popup"
  >
    <Bootstrap(Modal)>
      <Modal
        animation={true}
        autoFocus={true}
        backdrop={true}
        bsPrefix="modal"
        dialogAs={
          Object {
            "$$typeof": Symbol(react.forward_ref),
            "displayName": "ModalDialog",
            "render": [Function],
          }
        }
        enforceFocus={true}
        keyboard={true}
        manager={
          BootstrapModalManager {
            "adjustAndStore": [Function],
            "containers": Array [],
            "data": Array [],
            "handleContainerOverflow": true,
            "hideSiblingNodes": true,
            "modals": Array [],
            "restore": [Function],
            "scrollbarSize": 0,
          }
        }
        restoreFocus={true}
        show={false}
      >
        <Modal
          autoFocus={true}
          backdrop={true}
          backdropTransition={[Function]}
          className="modal"
          containerClassName="modal-open"
          enforceFocus={true}
          keyboard={true}
          manager={
            BootstrapModalManager {
              "adjustAndStore": [Function],
              "containers": Array [],
              "data": Array [],
              "handleContainerOverflow": true,
              "hideSiblingNodes": true,
              "modals": Array [],
              "restore": [Function],
              "scrollbarSize": 0,
            }
          }
          onClick={[Function]}
          onEnter={[Function]}
          onEntering={[Function]}
          onExited={[Function]}
          onHide={[Function]}
          onMouseUp={[Function]}
          renderBackdrop={[Function]}
          restoreFocus={true}
          role="dialog"
          show={false}
          style={Object {}}
          transition={[Function]}
        />
      </Modal>
    </Bootstrap(Modal)>
  </div>
</Popup>
`;
